AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -10,14 +10,24 @@
       "defaultValue": "",
       "allowedValues": [
         "eastus2euap",
-        "centraluseuap",
-        "centralus",
+        "northcentralus",
```

Comment:
Kaveesh - should we also be adding fairfax region (virginia) here, as same templates will be used there too..

Reply:
Makes sense, I'll update the regions to include virginia.

Reply:
updated

Reply:
@vishiy Can you please approve the PR again when you get a chance. It should be good to merge in now.